### ISSUE TITLE
Bind `qute.inlayHint.enabled` (true / false) with `editor.inlayHints.enabled` (on / off)

### ISSUE THREAD

**Issue by angelozerr** [xorye/vscode-quarkus#509]
According to https://code.visualstudio.com/updates/v1_67#_toggle-inlay-hints it exists a vsocde settings `editor.inlayHints.enabled`.

I wonder if it should be nice to bind this setting with our `qute.inlayHint.enabled`  ?

**Comment by rgrunber** [comment]
Makes sense `qute.inlayHint.enabled` & `qute.inlayHint.showSectionParameterType` should be disabled if `editor.inlayHints.enabled` is off. Otherwise, the value of those 2 should take over. Note that by default both of the qute settings are `true`.
